packages/server/test/e2e/5_headless_spec.ts

```diff
@@ -4,6 +4,30 @@ import Fixtures from '../support/helpers/fixtures'
 describe('e2e headless', function () {
   e2e.setup()
 
+  describe('ELECTRON_RUN_AS_NODE', () => {
+    beforeEach(function () {
+      process.env.ELECTRON_RUN_AS_NODE = 'true'
+    })
+
+    // cypress run --headless
+    e2e.it('tests in headless mode pass', {
+      spec: 'headless_spec.js',
```

Comment:
to debug this test project further, look in `packages/server/test/support/fixtures/projects/e2e`

Reply:
Thanks!